ios/DataLayer/Toolkits/AuthToolkit/Sources/AuthToolkit/Repositories/AuthRepository.swift

```diff
@@ -7,6 +7,7 @@ import DatabaseProvider
 import KeychainProvider
 import NetworkProvider
 import SharedDomain
+import DevstackKmpShared
 
 public struct AuthRepositoryImpl: AuthRepository {
```

Comment:
I would like to preserve an example of the old "iOS-native" errors. Can we use these shared errors only in Login and keep the old style in Registration? 🙂